cmd/snap/cmd_export_key.go

```diff
@@ -44,13 +49,46 @@ func init() {
 			return &cmdExportKey{}
 		}, map[string]string{
 			"account": i18n.G("Format public key material as a request for an account-key for this account-id"),
+			"revoke":  i18n.G("Export a revocation request for this account-key"),
 		}, []argDesc{{
 			name: i18n.G("<key-name>"),
 			desc: i18n.G("Name of key to export"),
 		}})
 	cmd.hidden = true
 }
 
+func fetchAccountKeyAssertion(sto *store.Store, pubKey asserts.PublicKey) (asserts.Assertion, error) {
```

Comment:
I'm not sure I understand why not simply doing a sto.Assertion instead of using a fetcher?